src/App.jsx

```diff
@@ -1,44 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActiveTab] = useState(tabs[0].id);
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+  // const handleTest = (id) => {
+  //   setActiveTab(id);
+  //   console.log('hey');
+  // }
```

Comment:
The `handleTest` function is commented out and not used in the current implementation. If this function is not needed, consider removing it to keep the code clean. If it's for testing purposes, ensure it's not left in the production code.